helix-term/src/commands/typed.rs

```diff
@@ -1162,6 +1163,22 @@ fn hsplit(
         }
     }
 
+    // check if there are views with height equal to 1
+    // if there are, close the newly created view
+    let mut reached_limit = false;
+    cx.editor.tree.views().for_each(|v| {
+        if v.0.area.height == 1 {
+            reached_limit = true;
+        }
+    });
+    if reached_limit {
```

Comment:
```suggestion
    if cx
        .editor
        .tree
        .views()
        .any(|(view, _focused)| view.area.height == 1)
    {
```

Reply:
Thanks :) . I tried to put all the code in the for_each loop at first but I got a double borrow error. I don't understand why it works with any() now.